### PR TITLE
Remove native test build from build.sh

### PR DESCRIPTION
**Pull request by jashook**
/cc @dotnet/coreclr-infra

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -893,10 +892,6 @@ while :; do
             __PartialNgen=1
             ;;
 
-        skiptests|-skiptests)
```
Comment (AaronRobinsonMSFT) [inline build.sh]: Are there any other scripts files that should be updated to remove this flags usage?

Comment (AaronRobinsonMSFT) [inline build.sh]: Are there any other scripts files that should be updated to remove this flags usage?

Comment (AaronRobinsonMSFT) [reply build.sh]: Please update the script documentation for the `-skiptests` flag.

Comment (elinor-fung) [reply build.sh]: Also the yml

**Pull request merged**